core/includes/form.inc

```diff
@@ -558,8 +558,10 @@ function form_get_cache($form_build_id, &$form_state) {
  * Stores a form in the cache.
  */
 function form_set_cache($form_build_id, $form, $form_state) {
-  // 6 hours cache life time for forms should be plenty.
-  $expire = 21600;
+  // The default cache_form expiration is 6 hours. On busy sites, the cache_form
+  // table can become very large. A shorter cache lifetime can help to keep the
+  // table's size under control.
+  $expire = config_get('system.core', 'form_cache_expiration');
```

Comment:
Related to my other note, in order for this to be set in settings.php we probably need something like this:

`$expire = settings_get('form_cache_expiration', config_get('system.core', 'form_cache_expiration'));`

Reply:
Good catch @herbdool 👍 ...I have moved the "generic" info in the docblock, and added a new comment to explain that we are attempting to read the setting from `settings.php` first.